format/nexus.py

```diff
@@ -287,10 +287,13 @@ def visit(self, nx_file, depends_on):
                     raise RuntimeError("Invalid units: %s" % units)
 
                 # is the axis moving? Check the values for this axis
-                v = item[()]
-                if min(v) < max(v):
-                    is_scan_axis = True
-                else:
+                v = item[...]
+                try:
+                    if min(v) < max(v):
+                        is_scan_axis = True
+                    else:
+                        is_scan_axis = False
+                except TypeError:
                     is_scan_axis = False
```

Comment:
I suggest this, which is safe if your dataset is scalar or iterable:
```suggestion
                v = item[...]
                if v.min() < v.max():
                    is_scan_axis = True
                else:
                    is_scan_axis = False
```